code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -20,6 +20,7 @@ var/global/list/image/splatter_cache=list()
 	var/basecolor="#A10808" // Color when wet.
 	var/list/datum/disease2/disease/virus2 = list()
 	var/amount = 5
+	appearance_flags = NO_CLIENT_COLOR
```

Comment:
there is really not a problem with this until we actually do have more color matricies